modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -412,6 +412,31 @@ def test_loc(data):
     with pytest.raises(KeyError):
         modin_df.loc["NO_EXIST"]
 
+    # From issue #3764
+    pandas_df = pandas.DataFrame([[1, 2, 3], [4, 5, 6]])
+    modin_df = pd.DataFrame([[1, 2, 3], [4, 5, 6]])
+
+    for left, right in [(2, 1), (6, 1), (lambda df: 70, 1), (90, 70)]:
+
+        def _test_loc_rows(df):
+            df.loc[left] = df.loc[right]
+            return df
```

Comment:
```suggestion
    def _test_loc_rows(df):
        df.loc[left] = df.loc[right]
        return df

    for left, right in [(2, 1), (6, 1), (lambda df: 70, 1), (90, 70)]:
```

I think it's clearer to move the function definition out of the loop.

Reply:
Done.